pico-slack.js

```diff
@@ -66,6 +67,24 @@ const processEvent = (event)=>{
 	return evt;
 };
 
+const handleSocketClose = ()=>{
+	Slack.connected = false;
+
+	// If the socket disconnects unexpectedly, try to reconnect.
+	if (!Slack.closing) reconnect();
```

Comment:
Thoughts on publishing `reconnect()` on the Slack object? Could randomly come in handy.

Reply:
Yeah, I was planning to just use the `connect` event for that, but you're right, it could be useful to tell the difference.

Reply:
Oh wait I misread this.  I thought you meant publishing an event.

Yeah, I'm into making it a public method also.  It's little more than some sugar on top of `Slack.connect()`, but I guess it doesn't hurt to be able to "refresh" a Slack connection at will.